xml/System.Reflection.Metadata/SignatureCallingConvention.xml

```diff
@@ -21,8 +21,7 @@
   </Base>
   <Docs>
     <summary>Specifies how arguments in a given signature are passed from the caller to the callee. The underlying values of the fields in this type correspond to the representation in the leading signature byte represented by a <see cref="T:System.Reflection.Metadata.SignatureHeader" /> structure.</summary>
-    <remarks>
-    </remarks>
+    <remarks>To be added.</remarks>
```

Comment:
Recommend removing the "TBA" text from the remarks section.

```suggestion
    <remarks></remarks>
```

Reply:
Feel free to ignore the suggestion if desired - I see that "TBA" text appears in other places in this method.